### PR TITLE
Change storage defaults

### PR DESCRIPTION
**Pull request by janezpodhostnik**
Change the storage parameters defaults to what was originally deployed.

- minimum account balance        =  0.001 FLOW
- Storage megabytes per FLOW  =  10.0

Also made it possible to set any `Storage megabytes per FLOW` on the bootstrap level.

### CODE REVIEW

**Review by Kay-Zee — APPROVED** [review]
I assume the changed root state commit is because the default values are used when generating these root state commits?

**Comment by janezpodhostnik** [comment]
> I assume the changed root state commit is because the default values are used when generating these root state commits?

Yes that is exactly it.

File: fvm/bootstrap.go [hunk 1]
```diff
@@ -89,6 +98,13 @@ func WithMinimumStorageReservation(reservation cadence.UFix64) BootstrapProcedur
 	}
 }
 
+func WithStoragePerFlow(ratio cadence.UFix64) BootstrapProcedureOption {
```
Comment (psiemens) [inline fvm/bootstrap.go:101]: I would add the units to be explicit. Also could be good to say `FLOW` to indicate that it's referring to the token.

```suggestion
func WithStorageMBPerFLOW(ratio cadence.UFix64) BootstrapProcedureOption {
```

**Review by psiemens — APPROVED** [review]
Looks good, just one minor comment.

File: fvm/bootstrap.go (hunk 1 above)
Comment (janezpodhostnik) [reply fvm/bootstrap.go]: fixed in 926ad3502784e8b51269df72f6f4ad89d698296d

**Pull request merged**